package.json

```diff
@@ -24,7 +24,7 @@
     "test:spec-min": "lerna run test:spec-min --no-bail",
     "test:spec-fast": "lerna run test:spec-fast --no-bail",
     "test:spec-main": "lerna run test:spec-main --no-bail",
-    "benchmark": "yarn benchmark:files 'packages/*/test/perf/**/*.test.ts'",
+    "benchmark": "yarn benchmark:files packages/*/test/perf/**/*.test.ts",
```

Comment:
Could you review what error happened without quotes? The quotes are there to prevent bash expansion and let mocha do it.

Reply:
Warning! Many benchmarks are missing now https://github.com/ChainSafe/lodestar/runs/4772657466?check_suite_focus=true

Reply:
so I added the single quotes back but I have to change the root tsconfig.json
```
"compilerOptions": {
  "typeRoots": ["node_modules/@types", "./packages/lodestar/types"],
}
```

@dapplion do you see any issues with it, is it possible/necessary to provide a separate tsconfig.json just for `yarn:benchmark`?

Reply:
Let's roll with this and if anyone has issues find another strategy